### DISCUSSION TITLE
Exeeded Quota, but still getting new Errors and Transaction

### DISCUSSION THREAD

**Discussion by crepitu** [getsentry/support#60] [Support] [answered]
My company currently still in Free plans, to see what can sentry do, but, after a while, the quota has been exceeded, but somehow we still getting new errors, as shown below
<img width="1440" alt="sentry1" src="https://user-images.githubusercontent.com/29648408/182748715-9f5b4432-7728-49e8-80c1-5fab1a5ac47f.png">
my question is, if we, decided to use buiesnies plan, and same problem occurs, will sentry charged us for those exceeded quota?

(edit)
I'v done a quick reaserch about this issue and did not see anything to help with the following question, sorry if I have missed somthing, my concerns If from this page https://help.sentry.io/account/billing/i-ve-exhausted-my-quota-what-now/
from those link, it tell us about " new events will not be accepted until the start of the next usage period", but in my case I still getting new accepted errors and transaction

**Comment by souredoutlook** [comment, marked as answer]
Hi there,

Without your org-slug it would be difficult to confirm, however, I believe what you're currently experiencing is something called a [Grace Period](https://help.sentry.io/account/billing/what-happens-when-i-run-out-of-event-capacity-and-a-grace-period-is-triggered/).

When the one-time grace period is triggered you should receive an email explaining this.

An important thing to note about the grace period:

>if you increase your capacity, events accepted during the grace period will be counted as consumed capacity.

Let me know if you want me to re-activate your ability to trial, this should unblock your event capacity. I'll need your org-slug to do this.
